### PR TITLE
Update README.md with debugging notes

### PR DESCRIPTION
**Pull request by ilyapuchka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,22 @@ Depending on the workflow of each team some of them may be disabled to suit thei
 ### Client app
 
 To install the client app:
- - build it in Xcode and copy to your applications directory
- - run `defaults write com.babylonhealth.WallEView Host <application address>` 
+ - build it in Xcode and copy to your applications directory or download the app attached to the latest GitHub release
```
Comment (danspndl) [inline README.md]:
```suggestion
 - build it in Xcode and copy to your applications directory or download the app attached to the [latest GitHub release](https://github.com/babylonhealth/Wall-E/releases)
```

File: README.md [hunk 2]
```diff
@@ -27,8 +27,22 @@ Depending on the workflow of each team some of them may be disabled to suit thei
...
 
 Iconography © https://dribbble.com/shots/2772860-WALL-E-Movie-Icons
+
+### Debugging
+
+Using https://dashboard.ngrok.com/get-started you can run the app locally and still get all incoming events from GitHub webhooks.
+
+- setup ngrok and start it, it will print out the public address from which all requests will be redirected to your localhost, i.e. https://randomnumber.ngrok.io
+
+- add a webhook to the repository where you want to test the app (https://github.com/babylonhealth/walle-debug):
```
Comment (danspndl) [inline README.md]:
```suggestion
- add a webhook to the repository where you want to test the app:
```

Comment (ilyapuchka) [reply README.md]: it's ok to mention it as we can use it for internal use, there is nothing "secret" there
```suggestion
- add a webhook to the repository where you want to test the app (https://github.com/babylonhealth/walle-debug for use by Babylon team members):
```

File: README.md [hunk 3]
```diff
@@ -27,8 +27,22 @@ Depending on the workflow of each team some of them may be disabled to suit thei
...
- - launch the app
+ - build WallEView.xcodeproj in Xcode and copy the app from build products directory to your applications directory or download the app attached to the [latest GitHub release](https://github.com/babylonhealth/Wall-E/releases)
+ - run `defaults write com.babylonhealth.WallEView Host <application address>` to set the url to the app
+ - launch the app and enjoy
 
 Iconography © https://dribbble.com/shots/2772860-WALL-E-Movie-Icons
+
+### Debugging
+
+Using https://dashboard.ngrok.com/get-started you can run the app locally and still get all incoming events from GitHub webhooks.
```
Comment (AliSoftware) [inline README.md]:
```suggestion
Using [the ngrok tool](https://dashboard.ngrok.com/get-started) you can run the app locally and still get all incoming events from GitHub webhooks.
```

**Review by AliSoftware — APPROVED** [review]
👍

File: README.md [hunk 4]
```diff
@@ -27,8 +27,22 @@ Depending on the workflow of each team some of them may be disabled to suit thei
 ### Client app
 
 To install the client app:
- - build it in Xcode and copy to your applications directory
- - run `defaults write com.babylonhealth.WallEView Host <application address>` 
- - launch the app
+ - build WallEView.xcodeproj in Xcode and copy the app from build products directory to your applications directory or download the app attached to the [latest GitHub release](https://github.com/babylonhealth/Wall-E/releases)
```
Comment (danspndl) [inline README.md]:
```suggestion
 - build `WallEView.xcodeproj` in Xcode and copy the app from build products directory to your applications directory or download the app attached to the [latest GitHub release](https://github.com/babylonhealth/Wall-E/releases)
```

File: README.md [hunk 5]
```diff
@@ -27,8 +27,22 @@ Depending on the workflow of each team some of them may be disabled to suit thei
...
 
 Iconography © https://dribbble.com/shots/2772860-WALL-E-Movie-Icons
+
+### Debugging
+
+Using [the ngrok tool](https://dashboard.ngrok.com/get-started) you can run the app locally and still get all incoming events from GitHub webhooks.
+
+- setup ngrok and start it, it will print out the public address from which all requests will be redirected to your localhost, i.e. https://randomnumber.ngrok.io
+
+- add a webhook to the repository where you want to test the app (https://github.com/babylonhealth/walle-debug for use by Babylon team members):
```
Comment (danspndl) [inline README.md:42]: The debug repo should be in our docs, IMO this should be kept generic

Comment (ilyapuchka) [reply README.md:42]: when we have it we can move it 👍

**Review by danspndl — APPROVED** [review]
👌

**Pull request merged**